core/src/services/azblob/backend.rs

```diff
@@ -68,6 +67,8 @@ pub struct AzblobBuilder {
     encryption_algorithm: Option<String>,
     sas_token: Option<String>,
     http_client: Option<HttpClient>,
+    /// batch_max_operations
```

Comment:
I don't think we need this line of comment

Reply:
fixed